VocaDb.ResXFileCodeGenerator/FileOptions.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace VocaDb.ResXFileCodeGenerator;
+
+public sealed record FileOptions
+{
+	public string InnerClassInstanceName { get; init; }
+	public string InnerClassName { get; init; }
+	public InnerClassVisibility InnerClassVisibility { get; init; }
+	public bool PartialClass { get; init; }
+	public bool StaticMembers { get; init; } = true;
+	public AdditionalText File { get; init; }
+	public string FilePath { get; init; }
+	public bool StaticClass { get; init; }
+	public bool NullForgivingOperators { get; init; }
+	public bool PublicClass { get; init; }
+	public string ClassName { get; init; }
+	public string? CustomToolNamespace { get; init; }
+	public string LocalNamespace { get; init; }
+	public bool Valid { get; init; }
+
+	public FileOptions()
+	{
+		LocalNamespace = "";
+		CustomToolNamespace = "";
+		ClassName = "";
+		InnerClassInstanceName = "";
+		InnerClassName = "";
+		File = null!;
+		FilePath = "";
```

Comment:
I'd replace these with `string.Empty`, but not mandatory.

Reply:
done